gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -950,4 +951,18 @@ def AbilitiesItemWindow ():
 	Window.ShowModal (MODAL_SHADOW_GRAY)
 	return
 
+def HookKeyPresses():
+	Window = GUIINV.InventoryWindow
+	Window.SetKeyPressEvent(SwitchPC)
+	return
+
+def SwitchPC(wIdx, key, mod):
+	if key >= 49 and key <= 54:
+		GemRB.GameSelectPCSingle(key - 48)
+		GUICommonWindows.SelectionChanged()
+		GUIINV.UpdateInventoryWindow ()
```

Comment:
running GUICommonWindows.RunSelectionChangeHandler () instead would allow for the function to be completely generic. So it can be moved to GUICommonWindows or GUICommon and reused in other modules with a portrait bar.